dal/dbConn.py

```diff
@@ -0,0 +1,28 @@
+from tinydb import TinyDB, Query
+
+db = TinyDB('db.json')
+
+
+def db_insert(name) -> None:
```

Comment:
@mansikulkarni96 : Could you make an abstract class with insert, update and delete operations. Then implement operations mentioned by subclass named RestDB. 
Make sure you have the get_connection functions too.

Reply:
refer: the following implementation
https://github.com/CentOS-PaaS-SIG/linchpin/tree/develop/linchpin/rundb 
@mansikulkarni96